website/docs/pact_broker/advanced_topics/provider_verification_badges.md

```diff
@@ -32,13 +32,31 @@ Requires version 2.62+ of the Pact Broker.
 
 Returns a status badge that can be displayed in a README file that indicates whether the latest version of a pacticipant with a specified tag can be deployed to the specified environment as identified by a tag.
 
-[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test/badge)](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test)
+[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production/badge)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production)
 
 ```text
 [![Can I deploy Foo status](https://your-broker/pacticipants/PACTICIPANT/latest-version/TAG/can-i-deploy/to/ENVIRIONMENT_TAG/badge)](https://your-broker/pacticipants/PACTICIPANT/latest-version/TAG/can-i-deploy/to/ENVIRIONMENT_TAG)
 ```
 
-[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test/badge?label=my+custom+label+here)](https://test.pactflow.io/pacticipants/FrontendWebsite/latest-version/test/can-i-deploy/to/test)
+[![Can I deploy Foo status](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production/badge?label=my+custom+label+here)](https://test.pactflow.io/pacticipants/pactflow-example-consumer/latest-version/prod/can-i-deploy/to/production)
+
+To set a custom label for the badge, set the `label` query parameter. eg `?label=my+custom+label+here`.
+
+## Can I Merge badge
+
+[![Can I Merge Status](/img/can-i-merge-badge.svg)](https://test.pactflow.io)
+
+### Using branches and environments
```

Comment:
Using branches.

environments not relevant to this section

maybe point to configuring main branch ref docs? https://docs.pact.io/pact_broker/branches#pacticipant-main-branch-property